packages/synapse-react-client/src/synapse-queries/dataaccess/useAccessRequirements.ts

```diff
@@ -163,24 +166,65 @@ export function useCreateLockAccessRequirement(
   })
 }
 
+function getAccessRequirementStatusQueryOptions<
```

Comment:
Pull out the `queryKey` and `queryFn` so we can use them in both `useQuery` (for fetching one) and `useQueries` (for fetching many)